package.json

```diff
@@ -63,7 +63,8 @@
       "json",
       "node",
       "ts"
-    ]
+    ],
+    "testRunner": "jest-jasmine2"
```

Comment:
We need to include the jasmine runner to be able to use `fail` in tests - its basically a bug in `jest-circus` - see https://github.com/facebook/jest/issues/11698